### ISSUE TITLE
Import error with snowpack and typescript

### ISSUE THREAD

**Issue by abumalick** [fullcalendar/fullcalendar#5881]
### Reduced Test Case

https://github.com/abumalick/snowpack-fullcalendar

### Bug Description

Using snowpack with typescript as a bundler, when I try the example React implementation of the library. I got an error:
```
Uncaught Error: Please import the top-level fullcalendar lib before attempting to import a plugin.
```

### Screenshots
![image](https://user-images.githubusercontent.com/8377185/95014938-3911ad00-0642-11eb-869f-1a2ca48857e1.png)

**Comment by abumalick** [comment]
https://github.com/fullcalendar/fullcalendar/blob/5187127f72b66dab67913f65f4fe2a64bbc98d46/packages/common/src/vdom.ts#L6

**Comment by abumalick** [comment]
I tested with create react app and typescript, it works normally. It must be related to the way snowpack works

**Comment by abumalick** [comment]
Opened a discussion at snowpack: https://github.com/pikapkg/snowpack/discussions/1207

**Comment by mlarue** [comment]
I noticed the same problem with snowpack and LitElement

**Comment by arshaw** [comment]
The snowpack discussion you link provides a workaround.

To fix the root problem, the fullcalendar packages shouldn't rely on import execution order. This will be something I implement in v6

**Comment by charkour** [comment]
The workout is not very clear, any suggestions? I have made a repo that reproduces the error here: [link](https://github.com/charkour/course-schedulizer-snowpack). The snowpack react app is in `client-course-schedulizer`. Clone the repo, navigate to the snowpack client app, `npm i && npm start`

**Comment by charkour** [comment]
I was able to resolve it by importing from `@fullcalendar/react/dist/vdom.js` rather than from the `core` package. 

```tsx
import React from "react";
import "@fullcalendar/react/dist/vdom.js";
import FullCalendar from "@fullcalendar/react";
import type { CalendarOptions } from "@fullcalendar/common";
import interactionPlugin from "@fullcalendar/interaction"; // needed for dayClick
import timeGridPlugin from "@fullcalendar/timegrid"

export const Calendar = (props: CalendarOptions) => {
  return (
    <>
      <FullCalendar {...props} />
    </>
  );
};
```

**Comment by mylastore** [comment]
> I was able to resolve it by importing from `@fullcalendar/react/dist/vdom.js` rather than from the `core` package.
> 
> ```tsx
> import React from "react";
> import "@fullcalendar/react/dist/vdom.js"; // KEY CHANGE
> import FullCalendar from "@fullcalendar/react";
> import type { CalendarOptions } from "@fullcalendar/common";
> import interactionPlugin from "@fullcalendar/interaction"; 
> import timeGridPlugin from "@fullcalendar/timegrid"
> 
> export const Calendar = (props: CalendarOptions) => {
>   return (
>     <>
>       <FullCalendar {...props} />
>     </>
>   );
> };
> ```

this work for me also with `@sveltejs/kit`

**Comment by marcoroth** [comment]
> `import '@fullcalendar/core/vdom.js'`

Thanks @mylastore! This also solved it for me for vanilla JS!

**Comment by arshaw** [comment]
Fixed in [v6.0.0](https://github.com/fullcalendar/fullcalendar/releases/tag/v6.0.0). The workaround is no longer necessary

**Comment by marcoroth** [comment]
I can confirm, this seems to be fixed in `v6.0.0`. Thanks @arshaw!
